### DISCUSSION TITLE
vite-proxy-config/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#69] [Announcements]
# vite-proxy-config/

我们在编写前端项目的时候，经常会遇到跨域的问题，当我们访问后端 API 的 URL 路径时，只要域名、端口或访问协议（如 HTTP 和 HTTPS）有一项不同，就会被浏览器认定为跨域。另外我们也会经常重复编写后端的域名，例如 https://example.com/api/some_end_point，https://example.com/api/other_end_point，针对这两种情况，可以直接用同一个配置来解决，即代理配置。

https://zxuqian.cn/vite-proxy-config/

**Comment by kortin99** [comment]
可以实现mock吗，类似webpack的before。针对开发环境下用户登录、用户信息、TOKEN等，不需要再起一个服务。
